contrib/internal/httputil/trace.go

```diff
@@ -43,7 +43,7 @@ func TraceAndServe(h http.Handler, w http.ResponseWriter, r *http.Request, servi
 
 	w = wrapResponseWriter(w, span)
 
-	if strings.EqualFold("true", os.Getenv("SIGNALFX_SERVER_TIMING_CONTEXT")) {
+	if v := os.Getenv("SIGNALFX_SERVER_TIMING_CONTEXT"); v == "" || v == "1" || strings.EqualFold(v, "true") {
```

Comment:
I think it should be:
```suggestion
if v := os.Getenv("SIGNALFX_SERVER_TIMING_CONTEXT"); v == "" || v != "0" || !strings.EqualFold(v, "false") {
```

do you agree @owais ?

Reply:
Do we have options or a config object in this repo? may move this condition there and have a check like `config.trace_response_header_enabled`.